ls1intum/Artemis
# `Programming exercises`: Update Java and Kolin templates to use Ares 1.8.0

### Checklist
#### General
#### Changes affecting Programming Exercises

### Motivation and Context

Ares 1.8.0 got released and it requires additional changes to the pom.xml tempates. This PR incorporates them. It also replaces some uses of the outdated name with `Ares`.

### Description
- Update to Ares 1.8.0
- Add Maven Enforcer Plugin
- Update Documentation

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Programming Exercise

Make sure that:
- New Programming exercises can be created and tests can be executed using the default template
- Also build the documentation and make sure that `AJTS` is not mentioned anymore.

### Review Progress

#### Code Review
#### Manual Tests
